discovery-handler-modules/debug-echo-discovery-handler/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "debug-echo-discovery-handler"
-version = "0.8.11"
+version = "0.8.12"
 authors = ["Kate Goldenring <kate.goldenring@microsoft.com>"]
 edition = "2018"
 rust-version = "1.58.1"
```

Comment:
The rust versions listed in all the cargo.tomls should be increased, as well.

Reply:
Good catch